target_chains/cosmwasm/contracts/pyth/src/state.rs

```diff
@@ -41,7 +41,7 @@ pub struct PythDataSource {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct ConfigInfo {
-    pub owner:                      Addr,
+    // TODO: ASK do we need to update config key?
```

Comment:
What do you mean here?

Reply:
I don't know to what format Cosmwasm serialized though but I think it will break. We can update the config and have proper migration or rename it to `deprecated_owner`  and keep it. @njk-64 what do you think?

Reply:
Yeah this will probably break. 

Actually, this brings up a good question, which is how are we going to manage changes to this config class going forward?

I think we probably want to (1) stick a version number field in ConfigInfo so we can implement hard migration logic, and (2) add `#repr[(C)]` so we can extend it with fields at the end without having rust be too smart about packing the fields of the struct and potentially reordering things.